@xen-orchestra/log/capture.test.js

```diff
@@ -0,0 +1,225 @@
+'use strict'
+
+const { describe, it, beforeEach } = require('node:test')
+const assert = require('node:assert/strict')
+const { createLogger } = require('@xen-orchestra/log')
+const { configure } = require('@xen-orchestra/log/configure')
+const { captureLogs } = require('@xen-orchestra/log/capture')
+const { createCaptureTransport } = require('@xen-orchestra/log/capture')
+const { dedupe } = require('@xen-orchestra/log/dedupe')
+
+describe('capture log', () => {
+  const logger = createLogger('test-logger')
+
+  const logsTransportDefault = []
+  const transportTest = message => {
+    logsTransportDefault.push(message)
+  }
+
+  const logsTransportFatal = []
+  const transportFatal = message => {
+    logsTransportFatal.push(message)
+  }
+
+  beforeEach(() => {
+    configure(
+      createCaptureTransport([
+        {
+          level: 'fatal',
+          transport: transportFatal,
+        },
+        {
+          level: 'debug',
+          transport: transportTest,
+        },
+      ])
+    )
+
+    logsTransportDefault.length = 0
+    logsTransportFatal.length = 0
+  })
+
+  it('should log test with all its attributes', () => {
```

Comment:
That's not what you are testing here: you are testing that logs not in `captureLogs()` are handled by the fallback transport.

Reply:
I tested what appeared in the Usage.md, here it was the "producer" part presenting the logger.